test/test_evaluation/test_train_evaluator.py

```diff
@@ -3081,6 +3111,36 @@ def test_eval_holdout_iterative_fit_no_timeout(self):
         self.assertEqual(rval[0]["status"], StatusType.DONOTADVANCE)
         self.assertEqual(rval[-1]["status"], StatusType.SUCCESS)
 
+    def test_eval_holdout_iterative_fit_no_timeout_multi_objective(self):
+        metrics = {
+            accuracy: 0.030303030303030276,
+            balanced_accuracy: 0.033333333333333326,
+        }
+        eval_iterative_holdout(
+            queue=self.queue,
+            port=self.port,
+            config=self.configuration,
+            backend=self.backend,
+            resampling_strategy="holdout",
+            resampling_strategy_args=None,
+            seed=1,
+            num_run=1,
+            scoring_functions=None,
+            output_y_hat_optimization=True,
+            include=None,
+            exclude=None,
+            disable_file_output=False,
+            instance=self.dataset_name,
+            metrics=list(metrics.keys()),
+            additional_components=dict(),
+        )
+        rval = read_queue(self.queue)
+        self.assertEqual(len(rval), 9)
+        for metric, loss in metrics.items():
+            self.assertAlmostEqual(rval[-1]["loss"][metric.name], loss)
+        self.assertEqual(rval[0]["status"], StatusType.DONOTADVANCE)
+        self.assertEqual(rval[-1]["status"], StatusType.SUCCESS)
```

Comment:
[Q]
Reading this seems to indicate that `rval[0]` is the most recent one and `rval[-1]` is the first evaluation? Just curious